data/json/monsters.json

```diff
@@ -4861,6 +4861,36 @@
       "FILTHY"
     ]
   },
+  {
+    "id": "mon_zombeaver",
+    "type": "MONSTER",
+    "name": "zombeaver",
+    "description": "This wall-eyed beast has a large wound on its side, and ribs with some flesh on them are seen through it.  The most prominent feature of this abomination is large incisors in its ominously open mouth, strange black goo is oozing out of it.  Seems like it isn't interested in trees anymore and is looking for some warm fresh to feed.",
+    "default_faction": "zombie",
+    "bodytype": "bear",
+    "species": [ "ZOMBIE" ],
+    "volume": "20250 ml",
+    "weight": 20000,
+    "hp": 30,
+    "speed": 80,
+    "material": [ "flesh" ],
+    "symbol": "r",
+    "color": "green",
+    "aggression": 100,
+    "morale": 100,
+    "melee_skill": 1,
+    "melee_dice": 1,
+    "melee_dice_sides": 6,
+    "melee_cut": 6,
+    "dodge": 2,
+    "vision_day": 30,
+    "vision_night": 5,
+    "harvest": "zombie_leather",
+    "anger_triggers": [ "PLAYER_CLOSE", "HURT" ],
```

Comment:
"anger_triggers": [ "PLAYER_WEARING_BIKINI" ]

Reply:
Actually, are those (and also the fire fear trigger) supposed to even matter for zombified animals? Zombies seem to be already permanently hostile to everything living nearby and probably aren't supposed to be scared of anything either.

Reply:
> Actually, are those (and also the fire fear trigger) supposed to even matter for zombified animals?

It seems that these anger triggers are remnants of some old system. Or just redundant and unnecessary fields that are safe to remove.

> Zombies seem to be already permanently hostile to everything living nearby

Not everything, they are neutral to arthropods and small animals, for example.

> probably aren't supposed to be scared of anything either.

Maybe blob cares for its host and doesn't want it to be burned or otherwise damaged.